methods/espresso/5.2.1.yml

```diff
@@ -16,3 +16,35 @@ Default:
       - path: /linalg/diag/none/davidson/none
 #      - regex: \/qm\/wf\/none\/tetrahedron\/.*
 #      - path: /opt/diff/ordern/cg/none
+GNU:
```

Comment:
@seankwarren: since the specifications are same in all cases, can we use `!include "methods/espresso/5.2.1.yml#/Default"`?

Reply:
trying to use a reference to an item inside the same file causes the build assets to fail, likely due to an infinite loop